viewer/index.html

```diff
@@ -27,58 +58,33 @@
           height: 100%;
           object-fit: contain;
       }
+      #navigation-buttons{
+        width: 100%;
```

Comment:
`width: 100%` not needed. And as you haven't changed box-model, it can cause bugs latter on.